react/graphql/recommendation.gql

```diff
@@ -95,6 +97,18 @@ query recommendation($input: RecommendationInput) {
               values
             }
           }
+          properties {
```

Comment:
Why not add this query to store-resources? We could use the [product fragments](https://github.com/vtex-apps/store-resources/tree/master/react/fragments)

Then we could have consistency with the product fields requested:
https://github.com/vtex-apps/store-resources/blob/master/react/queries/productSearchV3.gql#L39-L56

Reply:
Done!